blockchain/vm/evm.go

```diff
@@ -304,8 +304,10 @@ func (evm *EVM) CallCode(caller types.ContractRef, addr common.Address, input []
 	if evm.depth > int(params.CallCreateDepth) {
 		return nil, gas, ErrDepth // TODO-Klaytn-Issue615
 	}
-	// Fail if we're trying to transfer more than the available balance
-	if !evm.CanTransfer(evm.StateDB, caller.Address(), value) {
+	// Note although it's noop to transfer X ether to caller itself. But
+	// if caller doesn't have enough balance, it would be an error to allow
+	// over-charging itself. So the check here is necessary.
+	if !evm.Context.CanTransfer(evm.StateDB, caller.Address(), value) {
```

Comment:
How about making it evm.Context.CanTransfer or evm.CanTransfer to be consistent elsewhere(every callsite)?

Reply:
https://github.com/klaytn/klaytn/pull/1763/commits/b9af23c7d5af383f5657c40a6f7957c74a405455. Changed the `create` to use `evm.Context.CanTransfer`.